Geometry/TrackerGeometryBuilder/interface/pixelTopology.h

```diff
@@ -85,7 +112,7 @@ namespace phase1PixelTopology {
     return n / 2;
   }
 
-  constexpr uint32_t maxModuleStride = findMaxModuleStride();
+  constexpr uint16_t maxModuleStride = findMaxModuleStride();
```

Comment:
It is negligible actually, this was me thinking that such a small number was worth to be stored as `uint16` rather than `uint32` but I would revert it.